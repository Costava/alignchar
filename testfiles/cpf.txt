11111
22222
33333]
44444ff]
55555
66666    ]
77777y==]
88888
