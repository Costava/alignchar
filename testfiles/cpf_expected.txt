11111
22222
33333FFFF]
44444ffFF]
55555
66666    ]
77777y==F]
88888
